File: Examples/DotNET/SDK/VisualBasic/Hyperlink/GetParticularHyperlink.vb
Imports Aspose.Cloud

Namespace Hyperlink
    Class GetParticularHyperlink
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample-hyperlink.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Get a Particular Hyperlink from a Word
            Dim wordsHyperlinkResponse As WordsHyperlinkResponse = Common.WordsService.WordsHyperlinks.ReadDocumentHyperlinkByItsIndex(inputfile, 0, Common.FOLDER)

            Console.WriteLine((Convert.ToString(vbLf) & inputfile) + " have " + wordsHyperlinkResponse.Hyperlink.DisplayText + " hyperlink")
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Hyperlink/GetParticularHyperlink.cs
using System;
using Aspose.Cloud;

namespace CSharp.Hyperlink
{
    class GetParticularHyperlink
    {
        public static void Run()
        {
            string inputfile = "doc-sample-hyperlink.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);

            // Get a Particular Hyperlink from a Word
            WordsHyperlinkResponse wordsHyperlinkResponse = Common.WordsService.WordsHyperlinks.ReadDocumentHyperlinkByItsIndex(inputfile, 0, Common.FOLDER);

            Console.WriteLine("\n" + inputfile + " have " + wordsHyperlinkResponse.Hyperlink.DisplayText + " hyperlink");
        }
    }
}
